Makefile

```diff
@@ -37,6 +37,8 @@ MOCKERY_VERSION := 2.32.4
 MOCKERY_OUTPUT := $(shell $(INSTALL_PATH)/mockery --version 2>/dev/null)
 INSTALL_MOCKERY := $(findstring $(MOCKERY_VERSION),$(MOCKERY_OUTPUT))
 
+build_cardinal = OFF
```

Comment:
The parameter name is called "engine_type", which may be better. The options are "knowhere" or "cardinal".